tests/test_preupg.py

```diff
@@ -273,95 +274,125 @@ def test_solution_www_bold_tag(self):
 
 
 class TestScenario(base.TestCase):
-    temp_dir = None
-
-    def setUp(self):
-        self.temp_dir = tempfile.mkdtemp()
-
-    def tearDown(self):
-        shutil.rmtree(self.temp_dir)
-
+    '''
+    Test get_scenario method, to get directory with modules
+    '''
     def test_correct_content_scenario(self):
```

Comment:
Fixed in https://github.com/upgrades-migrations/preupgrade-assistant/pull/233/commits/107a550a8ec5aeb8ddcbd5e59fef0d6a99e3e0a2